deployment/components/common/worker-mounts.yaml

```diff
@@ -22,6 +22,9 @@
   - name: features-d
     hostPath:
       path: "/etc/kubernetes/node-feature-discovery/features.d/"
+  - name: kubelet-dir
+    hostPath:
+      path: "/var/lib/kubelet/config.yaml"
```

Comment:
@marquiz Do you have any suggestions for handling this part?
When a custom kubelet config path is used, I want to ensure the user-specified path (via the `kubelet-config-path` flag) is correctly propagated. There are two places where this path needs to be dynamic. 
1. when mounting the path so the NFD worker can read the file
2. when passing the path to the worker (this part is already handled).
The issue is that it's easy to parameterize this in Helm, but I haven't found a clean way to do it with Kustomize.

Also, for the Helm specifically, do you think it makes sense to place the flag under `worker.config`, or would another location be more appropriate?